packages/cli/src/util/telemetry/commands/microfrontends/pull.ts

```diff
@@ -0,0 +1,15 @@
+import { TelemetryClient } from '../..';
+import type { TelemetryMethods } from '../../types';
+import type { pullSubcommand } from '../../../../commands/microfrontends/command';
+
+export class MicrofrontendsPullTelemetryClient
+  extends TelemetryClient
+  implements TelemetryMethods<typeof pullSubcommand>
+{
+  trackCliOptionDpl(value: string | undefined) {
+    this.trackCliOption({
+      option: '--dpl',
+      value: value ?? '',
+    });
+  }
```

Comment:
- Prefer `dpl` to `--dpl` for option to match others
- Redact identifying info,  for ex: https://github.com/vercel/vercel/blob/a76debca06ee5b414941694687dd0ad98bca0de0/packages/cli/src/util/telemetry/commands/promote/index.ts#L9

```suggestion
  trackCliOptionDpl(value: string | undefined) {
    if (value) {
        this.trackCliOption({
	      option: 'dpl',
	      value: this.redactedValue,
	    });
    }
  }
```